lighthouse-core/report/v2/renderer/report-renderer.js

```diff
@@ -172,6 +172,9 @@ class ReportRenderer {
       element.appendChild(this._renderAudit(audit));
     }
 
+    // don't create a passed section if there are no passed
+    if (!passedAudits.length) return element;
```

Comment:
if categories get more stuff in them the early exit might not be the best for keeping track of what goes in and what doesn't, but good for now